shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/EraDefinitions.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE EmptyDataDecls #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Cardano.Ledger.EraDefinitions where
```

Comment:
Each era will make its own definition. If we wanted to move the Shelley definition higher, a better module name might be `Cardano.Ledger.Shelley`, which just declares the `Era` instance for Shelley.